wsgi/odyssy/events/models.py

```diff
@@ -9,22 +9,24 @@
 
 class Event(models.Model):
     name = models.CharField(max_length=30, default="Sample Name")
-    start_time = models.DateTimeField(default=datetime.datetime.now())
-    end_time = models.DateTimeField(default=datetime.datetime.now())
+    start_time = models.DateTimeField(default=datetime.datetime.now)
```

Comment:
No, it doesn't solves that issue and yes, I have tried using `timezone.now` generator.